src/features/Applet/Schedule/Calendar/Calendar.utils.tsx

```diff
@@ -7,61 +7,183 @@ import {
   HeaderProps,
   ToolbarProps,
 } from 'react-big-calendar';
-import { format } from 'date-fns';
+import { format, getISOWeek } from 'date-fns';
 
-import { Svg } from 'components';
+import { DateFormats } from 'consts';
 import i18n from 'i18n';
 import theme from 'styles/theme';
 import { variables } from 'styles/variables';
-import { StyledLabelBoldMedium } from 'styles/styledComponents/Typography';
-import { DateFormats } from 'consts';
 
 import { UiType } from './Event/Event.types';
 import { Toolbar } from './Toolbar';
 import { MonthHeader } from './MonthHeader';
 import { Event } from './Event';
 import { MonthView } from './MonthView';
 import { YearView } from './YearView';
-import { CalendarEvent, CalendarViews } from './Calendar.types';
-import { StyledTimeHeaderGutter } from './Calendar.styles';
+import { AllDayEventsVisible, CalendarEvent, CalendarViews } from './Calendar.types';
+import { mockedEvents } from './Calendar.const';
+import { TimeHeader, UiType as TimeHeaderUiType } from './TimeHeader';
+import { TimeGutterHeader } from './TimeGutterHeader';
+import { EventWrapper } from './EventWrapper';
+import { DateHeader } from './DateHeader';
 
 const { t } = i18n;
 
+export const getMoreText = () => `${t('more').toLowerCase()}...`;
+
 export const formatToYearMonthDate = (date?: Date) =>
   date && format(date, DateFormats.DayMonthYear);
 
+export const formatToWeekYear = (date: Date) => `${getISOWeek(date)} ${date.getFullYear()}`;
+
+export const allDayEventsSortedByDays = mockedEvents
+  .reduce(
+    (
+      acc: { date: string | undefined; eventsIds: { id: string; isHidden: boolean }[] }[],
+      event,
+    ) => {
+      const currentEventStartDate = formatToYearMonthDate(event.start);
+      if (event.allDayEvent || event.alwaysAvailable) {
+        if (acc.some((el) => el.date === currentEventStartDate)) {
+          acc.map((el) => {
+            if (el.date === currentEventStartDate) {
+              const ids = el.eventsIds;
+
+              return {
+                ...el,
+                eventsIds: ids.push({
+                  id: event.id,
+                  isHidden: ids.length > 2,
```

Comment:
maybe move this number (2) to constants?

Reply:
done